experimental/dds/tree2/src/feature-libraries/modular-schema/fieldChangeHandler.ts

```diff
@@ -189,10 +188,9 @@ export type NodeChangeRebaser = (
 	change: NodeChangeset | undefined,
 	baseChange: NodeChangeset | undefined,
 	/**
-	 * Deleted when the baseChange deletes the node, Revived when the baseChange revives the node.
-	 * Unchanged by default when undefined.
+	 * Whether or not the node is alive or dead in the input context of change
```

Comment:
How do we interpret undefined now?

Reply:
Undefined is equivalent to `Alive` now. I'll add that to the doc comments unless we think we should make this required.